templates/common/thumbnail_grid_item.html

```diff
@@ -52,10 +52,19 @@
       $elif webp_thumb is not None:
         <picture>
           <source type="image/webp" srcset="${webp_thumb['display_url']}" />
```

Comment:
The yall.js documentation suggests that this should be `data-srcset`:

```suggestion
           <source type="image/webp" data-srcset="${webp_thumb['display_url']}" />
```